src/config/config.ts

```diff
@@ -1 +1,20 @@
 export const db = 'mongodb://Admin:passwordforadmin531@ds163494.mlab.com:63494/car-finder';
```

Comment:
This should go the the env var and be defaulted to the local value like this:  db = process.env.DB_BLA_BLA || 'mongodb://...';

Reply:
Same everywhere

Reply:
Environment variables added only for production-mode.We are using config from the config object for tests.